shell/models/__tests__/management.cattle.io.node.ts

```diff
@@ -0,0 +1,82 @@
+import MgmtNode from '@shell/models/management.cattle.io.node';
+
+describe('class MgmtNode', () => {
+  const t = jest.fn(() => 'baz');
+  const ctx = { rootGetters: { 'i18n/t': t } };
+
+  const resetMocks = () => {
+    // Clear all mock function calls:
+    jest.clearAllMocks();
+  };
+
+  it('should not return addresses if they are not present in the resource status, the internalNodeStatus, or the rkeNode key in status', () => {
+    const mgmtNode = new MgmtNode({ status: {} }, ctx);
+
+    expect(mgmtNode.addresses).toStrictEqual([]);
+    resetMocks();
+  });
+  it('should return addresses if they are present directly on the resource status', () => {
```

Comment:
You have 2 tests where you repeat `should return addresses if they`. It should be contained in the same description having this part of text and remove it from the test. It will helps it also to be conciser.

Reply:
Done